<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EncryptionWindow</class>
 <widget class="QWizard" name="EncryptionWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Зашифровать</string>
  </property>
  <widget class="QWizardPage" name="change_file_or_message_encrypt">
   <attribute name="pageId">
    <string notr="true">1</string>
   </attribute>
   <widget class="QCheckBox" name="encrypt_file">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Зашифровать файл</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="encrypt_message">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Зашифровать сообщение</string>
    </property>
   </widget>
  </widget>
  <widget class="QWizardPage" name="insert_key">
   <attribute name="pageId">
    <string notr="true">3</string>
   </attribute>
   <widget class="Drop" name="name_file_key">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>391</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_isert_key">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>391</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_key">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>391</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>170</y>
      <width>158</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="button_ok_insert_key">
       <property name="text">
        <string>Принять</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_cancel_insert_key">
       <property name="text">
        <string>Сбросить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>391</width>
      <height>58</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="button_insert_file">
         <property name="text">
          <string>Вставить ключ из файла</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_insert_key">
         <property name="text">
          <string>Ввести ключ в поле</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="button_add_new_key">
       <property name="text">
        <string>Сгенерировать новый ключ</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWizardPage" name="result_operation">
   <attribute name="pageId">
    <string notr="true">4</string>
   </attribute>
   <widget class="QTextEdit" name="your_secret_message">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>311</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_encript_message">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>311</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Зашифровать сообщение</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_your_message">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>311</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ваше зашифрованное сообщение</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_encrypt_file">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>311</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Зашифровать файл</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>351</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="button_copy">
       <property name="text">
        <string>Скопировать сообщение</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_save_message_in_file">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Сохранить в файл</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWizardPage" name="enter_message_or_file_name">
   <attribute name="pageId">
    <string notr="true">2</string>
   </attribute>
   <widget class="QTextEdit" name="area_message">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>341</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="Drop" name="line_file_name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>341</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_line_file_name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>331</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Перетащите сюда файл который Вы хотите зашифровать</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_area_message">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите ниже сообщение которое Вы хотите зашифровать</string>
    </property>
   </widget>
   <zorder>line_file_name</zorder>
   <zorder>area_message</zorder>
   <zorder>label_line_file_name</zorder>
   <zorder>label_area_message</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Drop</class>
   <extends>QLineEdit</extends>
   <header>drop.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
